projects/website/generate-news.js

```diff
@@ -105,11 +105,7 @@ function generateRoutes() {
     }
   });
   var final_routes = ROUTES_TEMPLATE.replace('${routes}', routes);
-  fs.writeFileSync(
-    FINAL_ROUTES_PATH,
-    `// tslint:disable
-  ${final_routes}`
-  );
+  fs.writeFileSync(FINAL_ROUTES_PATH, `// tslint:disable\n${final_routes}`.trim());
```

Comment:
This looks like just a formatting change. Did something change between branches that made prettier re-format this line?